tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -799,3 +790,123 @@ def test_comment_reactions_task_api_error_404(self, mock_metrics, get_jwt):
         self.comment.refresh_from_db()
         assert self.comment.reactions is None
         mock_metrics.incr.assert_called_with("github_pr_comment.comment_reactions.not_found_error")
+
+
+@region_silo_test(stable=True)
+class TestSafeForComment(GithubCommentTestCase):
+    base_url = "https://api.github.com"
+
+    def setUp(self):
+        super().setUp()
+        access_token = "xxxxx-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxxxx"
+        expires_at = isoformat_z(timezone.now() + timedelta(days=365))
+        self.pr = self.create_pr_issues()
+        self.mock_metrics = patch("sentry.tasks.integrations.github.pr_comment.metrics").start()
+        self.gh_path = self.base_url + "/repos/getsentry/sentry/pulls/{pull_number}"
+        installation = self.integration.get_installation(organization_id=self.organization.id)
+        self.gh_client = installation.get_client()
+        responses.add(
+            responses.POST,
+            self.base_url
+            + f"/app/installations/{self.gh_client._get_installation_id()}/access_tokens",
+            json={"token": access_token, "expires_at": expires_at},
+        )
+
+    @responses.activate
+    def test_simple(self):
+        responses.add(
+            responses.GET,
+            self.gh_path.format(pull_number=self.pr.key),
+            status=200,
+            json={"changed_files": 5, "additions": 100, "deletions": 100},
+        )
+
+        assert safe_for_comment(self.gh_client, self.gh_repo, self.pr)
+
+    @responses.activate
+    def test_error__rate_limited(self):
```

Comment:
Is there a convention for this double underscore syntax? Haven't seen it before

Reply:
No convention AFAIK, I just thought it made the test names seem more organized.

Reply:
I saw someone else do this in a PR recently (I think Cathy?)